pkg/services/hcloud/server/server.go

```diff
@@ -131,23 +142,39 @@ func (s *Service) Reconcile(ctx context.Context) (res reconcile.Result, err erro
 
 	// check whether server is attached to the network
 	if err := s.reconcileNetworkAttachment(ctx, server); err != nil {
-		return res, fmt.Errorf("failed to reconcile network attachement: %w", err)
+		reterr := fmt.Errorf("failed to reconcile network attachement: %w", err)
```

Comment:
Typo: attachement --> attachment (occurs several times).

Reply:
Thanks! That has been in there for a while